src/components/ProjectsList/__tests__/ProjectsList.test.tsx

```diff
@@ -1,13 +1,33 @@
 import { ProjectsList } from '../ProjectsList';
 import { MemoryRouter } from 'react-router-dom';
-import { render } from '@testing-library/react';
-
-// temporarily disabled
-test('renders ProjectsList', () => {
-  render(
-    <></>
-    // <MemoryRouter>
-    //   <ProjectsList />
-    // </MemoryRouter>
-  );
+import { render, screen } from '@testing-library/react';
+
+describe('display project list page', () => {
```

Comment:
I think `'ProjectList component'` description is more appropriate, at least `page` should not be there as it's just component in our terminology